layout/pages/settings/interface.xml

```diff
@@ -82,6 +84,23 @@
 					</Button>
 				</ChaosSettingsEnum>
 
+				<ChaosSettingsEnum text="Playtest consent" infomessage="Playtest consent" convar="cl_playtest_enabled" hasdocspage="false" tags="consent">
+					<RadioButton group="enableconsent" text="#Common_Off" value="0" />
+					<RadioButton group="enableconsent" text="#Common_On" value="1" />
+				</ChaosSettingsEnum>
+
+				<ChaosSettingsEnum
+					id="PlaytestDOSARemove"
+					text="Playtest DOSA remove"
+					infomessage="Remove DOSA from playtest consent (For development purposes)"
+					hasdocspage="false"
+					tags="dosa"
+				>
+					<Button class="button horizontal-align-right" onactivate="SettingsShared.PlayTestDosaRemove()">
+						<Label class="button__text" text="Remove DOSA" />
+					</Button>
+				</ChaosSettingsEnum>
```

Comment:
Since this setting is for development purposes, it should be either be removed or hidden if the game isn't run in developer mode. You could wrap this in a `ConVarEnabler` panel pointed at the `developer` cvar to hide it.